### DISCUSSION TITLE
1.3.4 + Bltouch + Octoprint

### DISCUSSION THREAD

**Discussion by LeJackal78** [Jyers/Marlin#1043] [Q&A] [answered]
I will first say I am a complete newb, to 3d printing, and have 0 clue on compiling anything. I had been having issues with wierd firmware crashes with my bltouch so when I came across this firmware and several youtubers I watch discussed it I decided to give it a try. I got it patched and everything running great until I tried to connect with octoprint I got a M112 error, So i found the help on it made the changes that was recommened about disconnecting over the error. However I still can not get octoprint to connect it no longer gives an error just times out. I don't know if the meat packer issue is still a thing but I am clueless on how to compile the firmware once disabling it anyway and don't even know where to begin. I am hoping someone will be able to point me in the right direction as I really like the functionality of this firmware and possiblity of having the screen update with prints sent by octoprint. I jsut can't get it to connect

also this is the end of my log file after trying to connect.................

2021-06-21 16:00:28,292 - octoprint.plugins.action_command_notification - INFO - Notifications cleared
2021-06-21 16:00:31,464 - octoprint.util.comm - INFO - Changing monitoring state from "Offline" to "Opening serial connection"
2021-06-21 16:00:31,465 - octoprint.util.comm - INFO - Connecting to port /dev/ttyUSB0, baudrate 9600
2021-06-21 16:00:31,488 - octoprint.util.comm - INFO - Changing monitoring state from "Opening serial connection" to "Connecting"
2021-06-21 16:00:31,490 - octoprint.util.comm - INFO - M110 detected, setting current line number to 0
2021-06-21 16:00:53,450 - octoprint.server.util.sockjs - INFO - Client connection closed: ::ffff:192.168.50.76

**Comment by Jyers** [comment, marked as answer]
Have you tried setting the baudrate to 115200? it seems there that it's trying to connect at 9600

**Reply by LeJackal78** [reply]
Thank you this worked, I had tried several different rates and none of them worked, but this one did...again thank you

**Comment by smooreace** [comment]
First... Use the Wiki! Not sure what process you followed to " the changes
that was recommened about disconnecting over the error", but all thats
needed is shown in the Wiki. Almost more importantly, all USB cables
aren't the same, which is astoundingly pathetic! USB is a standard. Varying
from that standard makes it no longer a USB cable, but I digress. Try using
a QUALITY cable that actually has a data speed rating in its specs.

**Reply by LeJackal78** [reply]
I had already bought a good USB cable for it, and the instructions i was refering to was the what to do when printer reports error setting, told it to stop print but stay connected. as well as add the gcode for the before and after print stuff for the control panel to show the information from the print. However I tried the baudrate change that Jyers said and it worked perfectly I had already tried a few different ones but not that one...Thank you though for replying with ideas to try!

**Reply by smooreace** [reply]
Great... it was easy to assume what you did. But it was safer to just put it out there.
